src/addons/addonmessage.cpp

```diff
@@ -61,6 +61,9 @@ Addon* AddonMessage::create(QObject* parent, const QString& manifestFileName,
   message->m_date = messageObj["date"].toInteger();
   message->planDateRetranslation();
 
+  // if "notify" is not specified in the manifest, default to true
+  message->m_shouldNotify = messageObj["notify"].toBool(true);
```

Comment:
Given this is a new API and the previous behavior used to be not showin gnotifications, should the default here be false?

Reply:
the previous behavior was to show notifications

Reply:
LOL, I retrieve my ~dumb~ comment.